app.js

```diff
@@ -5,7 +5,33 @@ var port = process.env.PORT || 4000;
 
 app.get('/', function(req, res) {
     //res.send('<html><head></head><body><h1>Teams:</h1></body></html>');
-    res.send(['acs-cognitive-team', 'acs-fabric-team', 'acs-appdev-team', 'acs-database-team', 'acs-database-db2', 'acs-compute-team', 'acs-security-team', 'acs-network-team', 'acs-storage-team', 'acs-virt-team', 'acsvpc']);
+    res.send(['acs-cognitive-team', 'acs-fabric-team', 'acs-appdev-team', 'acs-database-team', 'acs-database-db2', 'acs-compute-team', 'acs-security-team', 'acs-network-team', 'acs-storage-team', 'acs-virt-team', 'acsvpc', 'acs-paascore']);
```

Comment:
I don't think we'll need to make use of the base route `https/yourapp.com/` so I think we can delete this route `app.get('/', function () {} )` from our app